tests/test_common.h

```diff
@@ -825,6 +827,7 @@ class TestSuite {
                (cur_test->options.printTestMessage || cur_test->displayMsg) &&
                !cur_test->suppressMsg ) ||
              globalMsgFlag() ) {
+            _fix_format(format);
```

Comment:
Unlike the logger in NuRaft, both `_msg` and `_msgt` are just wrappers around traditional `printf` so it should behave exactly the same as `printf` without `_fix_format`. Instead, please refer my comment below.